test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Alias.Tests.ps1

```diff
@@ -211,3 +211,25 @@ Describe "Get-Alias" -Tags "CI" {
         }
     }
 }
+
+Describe "Get-Alias null tests" -Tags "CI" {
+
+  $testCases = 
+    @{ data = $null; value = 'null' },
+    @{ data = [String]::Empty; value = 'empty string' }
+
+  Context 'Check null or empty value to the -Name parameter' {
+    It 'Should throw if <value> is passed to -Name parameter' -TestCases $testCases {
+      param($data)
+      try {Get-Alias -Name $data }
+      catch [System.Management.Automation.ParameterBindingException] {$_.Exception.ErrorId | Should Be 'ParameterArgumentValidationError'}
```

Comment:
Please, use 

```powershell
 try
    {
        get-item "ThisFileCannotPossiblyExist" -ErrorAction Stop
        throw "No Exception!"
    }
    catch
    {
        $_.FullyQualifiedErrorId | should be "PathNotFound,Microsoft.PowerShell.Commands.GetItemCommand"
    }
```

pattern, as described in https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md